src/options/options.ts

```diff
@@ -8,19 +7,23 @@ import {UIPPlugin} from '../core/plugin';
 export class UIPOptions extends UIPPlugin {
   static is = 'uip-options';
 
-  @attr({defaultValue: 'vertical'}) public mode: string;
-  @attr({defaultValue: 'uip-light'}) public theme: string;
-
+  private _mode: string;
+  private _theme: string;
+  
```

Comment:
In my current implementation after root attributes change in options fires checkMarker() and updates current mode/theme. So we don`t have previous state in what we can return after resizing. Actually it applies only mode option, but I`ve done similar for both attributes to avoid confusion.
Maybe I didn`n correctly understand proposition?